### PR TITLE
Delete tags

### PR DESCRIPTION
**Pull request by benlovell**
Gives the resource manager the ability to delete tags.

### CODE REVIEW

File: app/views/tags/index.html.erb [hunk 1]
```diff
@@ -16,12 +16,19 @@
...
           <tr class="table-header">
             <th>Name</th>
+            <th></th>
           </tr>
         </thead>
         <tbody>
           <% @tags.each do |tag| %>
             <tr class="t-tag">
               <td><%= tag.name %></td>
+              <td><%= link_to 'Delete',
```
Comment (benbarnett) [inline app/views/tags/index.html.erb]: Just had a quick chat here about 'tags' vs 'groups'. The users of this system have been shown a number of iterations of the prototype that refer to this functionality as groups...

Could we stick with that for anything in the UI? I guess it's fine to have them described as tags in the model, but their understanding on the domain language is a 'group' for this kind of thing.

Comment (benlovell) [inline app/views/tags/index.html.erb]: That's cool with me. Although I'm going to go the whole 🐷 and rename them everywhere.

File: app/views/tags/index.html.erb [hunk 2]
```diff
@@ -16,12 +16,19 @@
         <thead>
           <tr class="table-header">
             <th>Name</th>
+            <th></th>
```
Comment (Guntrisoft) [inline app/views/tags/index.html.erb]: I've been putting 'Actions' in here as the header - it's wise to have a header for accessibility even if you wrap it in span with a sr-only class to only show it to screen readers.

**Comment by benlovell** [comment]
This will be superseded by an incoming PR.

**Pull request closed without merging**